src/networks.json

```diff
@@ -1008,6 +1008,20 @@
     "start": 877115,
     "logo": "ipfs://bafkreigovfh3pinsdih777issfgaflwu2yjzroljs2642gbvwikcd3nm4i"
   },
+  "2020": {
+    "key": "2020",
+    "name": "Public Mint",
+    "shortName": "publicmint",
+    "chainId": 2020,
+    "network": "mainnet",
+    "multicall": "0x8De9F846d72Bd5CE7BC572994E5420d6CB047bF4",
```

Comment:
Can verify and publish the contract code here https://explorer.publicmint.io/address/0x8De9F846d72Bd5CE7BC572994E5420d6CB047bF4/contracts

Reply:
Contract verified and published.